sunpy/time/tests/test_utime.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import absolute_import, division, print_function
+
+from astropy.time import Time
+
+
+def test_utime_t0():
+    assert Time('1979-01-01T00:00:00').utime == 0.0
+
+
+def test_utime_random_date():
+    assert Time('2018-01-13T13:32:56').utime == 1231853576.0
```

Comment:
Could you please add a test which is marked with `@pytest.mark.xfail` of the reverse conversion? i.e. the open astropy issue?